presto-accumulo/src/test/java/io/prestosql/plugin/accumulo/AccumuloQueryRunner.java

```diff
@@ -159,8 +159,8 @@ public static Session createSession()
     }
 
     /**
-     * Gets the AccumuloConnector singleton, starting the MiniAccumuloCluster on initialization.
-     * This singleton instance is required so all test cases access the same MiniAccumuloCluster.
+     * Gets the AccumuloConnector singleton, starting the Accumulo Docker container on initialization.
+     * This singleton instance is required so all test cases access the same Accumulo
```

Comment:
Do we still need a singleton? It would be nice to have it as regular closeable resource instead.

Reply:
No, not needed. I can create a `TestingAccumuloServer` and refactor the tests to use it to be more similar to how the other connectors work.

Reply:
Turns out it is needed after all due to the fixed ports, multiple containers try to spawn at the same time and it fails. If I try a lock approach so only one container runs at a time, one of the test framework classes (distributed/integration smoke test) inexplicable fail because they are unable to connect to Accumulo once a new container is spawned.

I suggest with stick with the Singleton -- I'll still refactor into a `TestingAccumuloServer` for cleanliness.